packages/mocks/package.json

```diff
@@ -10,31 +10,15 @@
 
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
-  "exports": {
-    "./package.json": "./package.json",
-    ".": {
-      "types": "./dist/index.d.ts",
-      "default": "./dist/index.js"
-    },
-    "./consoleHandlers": {
-      "types": "./dist/consoleHandlers/index.d.ts",
-      "default": "./dist/consoleHandlers/index.js"
-    }
-  },
 
   "scripts": {
     "clean": "rimraf ./dist ./coverage tsconfig.tsbuildinfo",
     "clean:all": "npm run clean -- ./node_modules ./.rollup.cache",
-    "build": "npm run clean && npm run compile",
-    "compile": "tsc --build --verbose",
+    "build": "rollup -c --bundleConfigAsCjs",
     "lint": "eslint . ",
     "lint:fix": "eslint . --fix"
   },
   "dependencies": {
-    "@kubev2v/legacy": "*",
```

Comment:
@sjd78  added this as part of TS project references - is it not needed anymore?

Reply:
currently we don't use this package, we can add it when we need to.

p.s.
i'm not sure we should use it, we will just need to remove it in a few PRs